### ISSUE TITLE
Error in .svg display when converting a README.md to main page in html

### ISSUE THREAD

**Issue by MikaelSlevinsky** [doxygen/doxygen#6978]
The issue can be seen at the top of the main page https://mikaelslevinsky.github.io/FastTransforms/

The two build badges are in fact URLs that have an svg file embedded in the website. However, the third badge (problematic) is a URL that ends with .svg.
Markdown:
```
[![Travis](https://travis-ci.org/MikaelSlevinsky/FastTransforms.svg?branch=master)](https://travis-ci.org/MikaelSlevinsky/FastTransforms) [![Build status](https://ci.appveyor.com/api/projects/status/er98t0q3bsx4a5l9/branch/master?svg=true)](https://ci.appveyor.com/project/MikaelSlevinsky/fasttransforms/branch/master) [![](https://img.shields.io/badge/docs-master-blue.svg)](https://MikaelSlevinsky.github.io/FastTransforms)
```
Doxygen-1.8.15 converted html:
```
<p><a href="https://travis-ci.org/MikaelSlevinsky/FastTransforms"><img src="https://travis-ci.org/MikaelSlevinsky/FastTransforms.svg?branch=master" alt="Travis" class="inline"/>
</a> <a href="https://ci.appveyor.com/project/MikaelSlevinsky/fasttransforms/branch/master"><img src="https://ci.appveyor.com/api/projects/status/er98t0q3bsx4a5l9/branch/master?svg=true" alt="Build status" class="inline"/>
</a> <a href="https://MikaelSlevinsky.github.io/FastTransforms"><object type="image/svg+xml" data="https://img.shields.io/badge/docs-master-blue.svg"></object>
/>
</a></p>
```
Doxygen decides to convert the .svg image differently when the badge has this identifier in the URL, choosing to represent it as an `object` instead of an `img`, and appending an extra `/>`, which remains visible in the main page.

One possible desired output would be:
```
<p><a href="https://travis-ci.org/MikaelSlevinsky/FastTransforms"><img src="https://travis-ci.org/MikaelSlevinsky/FastTransforms.svg?branch=master" alt="Travis" class="inline"/>
</a> <a href="https://ci.appveyor.com/project/MikaelSlevinsky/fasttransforms/branch/master"><img src="https://ci.appveyor.com/api/projects/status/er98t0q3bsx4a5l9/branch/master?svg=true" alt="Build status" class="inline"/>
</a> <a href="https://MikaelSlevinsky.github.io/FastTransforms"><img src="https://img.shields.io/badge/docs-master-blue.svg" alt="" class="inline"/>
</a></p>
```

**Comment by albert-github** [comment]
I think this problem has been addressed in pul request #6914 please have a look at it.
Furthermore:
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem?  Please don't add external links as they might not be persistent.

**Comment by MikaelSlevinsky** [comment]
Thanks for referencing #6914; it appears to be the same issue.

**Comment by albert-github** [comment]
Code (of pull request #6914) has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
